packages/apps/explorer/src/components/layout/layout.tsx

```diff
@@ -14,7 +14,6 @@ export const Layout: FC<IProps> = ({ children }: IProps) => {
   return (
     <div className={documentStyle}>
       <Header />
-      <Stack marginBlock="xl" />
       <Stack
```

Comment:
you can add the marginblockstart to the below stack

Reply:
Isa mentioned that the header height isn't tokenized...